qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -107,6 +107,17 @@
   </screenshots>
 
   <releases>
+   <release version="2022.09.22-3" date="2022-09-22">
+     <description>
+       <ul>
+         <li>New OSM data as of September 12, 2022</li>
+         <li>Fixed wrong elevation data between -1,-1 and 1,1 latitudes and longitudes</li>
```

Comment:
• New OSM maps data as of September 12
• Fixed wrong elevation data between -1,-1 and 1,1 latitudes and longitudes
• Fixed crash on startup after editing some OSM objects
• Bookmark lists are now sorted by recently modified time (sorting by name is planned!)
• Improved displaying of the next street name directions
• Show features in the viewport at the top of search results
• Significantly speedup some search scenarios
• Improved category ranking & address search.
• Fixed highlighted hotel search results icon on the map
• Improved German category search
• Added driving_school, music_school, language_school, archaeological_site, craft-painter
• Removed backgrounds from historic cemetery icons
• Added different castle types: castrum, fortified_church, fortress, hillfort, kremlin, manor, palace & shiro
• Increased visibility of education & some sports
• Separated furniture & kitchen types
• Better looking leisure=track
• Fixed vending=fuel icon

Reply:
Thanks!